Doc/tutorial/appendix.rst

```diff
@@ -31,6 +31,32 @@ Typing an interrupt while a command is executing raises the
 :exc:`KeyboardInterrupt` exception, which may be handled by a :keyword:`try`
 statement.
 
+.. _tut-var:
+
+Special ``_`` variable
+-------------------------
+
+There is a special variable ``_`` in interactive mode, stored in the :mod:`builtins` 
```

Comment:
Is it really true that _ is stored in builtins? I played around with it a while, and couldn't verify that. And even if it's true, is it important? I don't think so. Do you have another reference to _ being in builtins?

Reply:
This was stated in bpo-41394 too.
https://docs.python.org/3/reference/lexical_analysis.html#reserved-classes-of-identifiers

Reply:
Interesting. I still think it's unimportant for the tutorial and should be removed.

Reply:
OK. Working on it.